frontend/src/scenes/session-recordings/player/sessionRecordingPlayerLogic.ts

```diff
@@ -1307,6 +1309,23 @@ export const sessionRecordingPlayerLogic = kea<sessionRecordingPlayerLogicType>(
                 height: parseFloat(iframe.height),
             })
         },
+        takeScreenshot: async () => {
+            actions.setPause()
```

Comment:
you can do a toast with a status - it's how we do the export toast - so that you can start a "taking screenshot spinner" toast and users get immediate feedback

there was a long enough pause when I clicked that I wondered if i hadn't clicked

Reply:
yep, here's how you start a toast passing a promise to it, so it stays until the promise resolves

https://github.com/PostHog/posthog/blob/26bf7585973a5f5da42b17b7320d045e082625d4/frontend/src/lib/components/ExportButton/exportsLogic.ts#L136-L140

can we do that and then 100% ship this, it's cool